src/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -632,6 +632,7 @@
     <Reference Include="System.Security.Cryptography.Algorithms" />
     <Reference Include="System.Security.Cryptography.Encoding" />
     <Reference Include="System.Security.Cryptography.Primitives" />
+    <Reference Include="System.IO.FileSystem" />
```

Comment:
nit: move this line up to be alphabetical ordering (i.e. above System.Security.Cryptography.Algorithms)